lib/active_record/json_validator/validator.rb

```diff
@@ -2,6 +2,7 @@ class JsonValidator < ActiveModel::EachValidator
   def initialize(options)
     options.reverse_merge!(message: :invalid_json)
     options.reverse_merge!(schema: nil)
+    options.reverse_merge!(validator_options: {})
```

Comment:
Because those are option we pass to `::JSON::Validator.fully_validate`. I guess we could use `:options` too.